book/src/generate_docs.md

```diff
@@ -3,7 +3,7 @@
 And finally the last feature! Just run the following command (note the `-m doc` at the end):
 
 ```console
-> cargo run --release -- -c YourGirFile.toml -d ../gir-files --doc-target-path the-output-file-name -m doc
+> gir -c Gir.toml -d ../gir-files --doc-target-path docs.md -m doc
```

Comment:
```suggestion
cargo run --release -- -c Gir.toml -d ../gir-files --doc-target-path docs.md -m doc
```

Reply:
If I run your suggested command, I get ``error: a bin target must be available for `cargo run` ``. In the previous commands of the tutorial, gir was also used instead of using the `cargo run` command. That is why I suggested the above change.